src/model/element-base.ts

```diff
@@ -105,6 +106,25 @@ export interface ElementBaseInit extends ClassInit {
   readonly slots?: Slot[];
 }
 
+/**
+ * The element's runtime contents.
+ */
+export type ElementTemplate = {
+  /**
+   * HTML that is stamped out without data binding or other
```

Comment:
do we actually have this anywhere?

Reply:
Not yet, but I wanted to have a type for when we do add this.